kostyor/conf.ini

```diff
@@ -1,3 +1,4 @@
 [global]
 host = 127.0.0.1
 port = 5000
+discovery_methods = first,second
```

Comment:
You're welcome :)

29 июля 2016 г. 11:25 пользователь "Julia Varlamova" <
notifications@github.com> написал:

> In kostyor/conf.ini
> <https://github.com/sc68cal/Kostyor/pull/12#discussion_r72757326>:
>
> > @@ -1,3 +1,4 @@
> >  [global]
> >  host = 127.0.0.1
> >  port = 5000
> > +discovery_methods = first,second
>
> Thank you for explanation!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/sc68cal/Kostyor/pull/12/files/5c8dab03b11bf5d0f61301bbb3595efe5be574b5#r72757326>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEzEDWk7jsaKzZM_KUeNLhnMmFJhs084ks5qabj-gaJpZM4JQxHZ>
> .
>